### PR TITLE
[ADD] legalsylvain + pedrobaeza + MiquelRForgeFlow as PSC representative of OpenUpgrade project

### PR DESCRIPTION
**Pull request by legalsylvain**
CC : @pedrobaeza, @StefanRijnhart, @MiquelRForgeFlow, @hbrunn

### CODE REVIEW

File: conf/psc/openupgrade.yml [hunk 1]
```diff
@@ -13,5 +13,6 @@ openupgrade-maintainers:
 openupgrade-maintainers-psc-representative:
   members:
     - StefanRijnhart
+    - legalsylvain
```
Comment (pedrobaeza) [inline conf/psc/openupgrade.yml:16]: Please include me to be able also to handle it, or I think I will lose it the current right when synced

Comment (legalsylvain) [reply conf/psc/openupgrade.yml:16]: I'm not sure how this repo works, but I _think_ that as you are member of the core team (https://github.com/OCA/repo-maintainer-conf/blob/master/conf/psc/core.yml) you have very broad rights granted by this "global rule (https://github.com/OCA/repo-maintainer-conf/blob/master/conf/global.yml)

Anyway updated.

Comment (MiquelRForgeFlow) [reply conf/psc/openupgrade.yml:16]: add me too :)

Comment (legalsylvain) [reply conf/psc/openupgrade.yml:16]: done.

**Review by pedrobaeza — APPROVED** [review]

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by rvalyi — APPROVED** [review]

**Pull request merged**